tests/examples/python/yaml-test/__main__.py

```diff
@@ -11,11 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from pulumi_kubernetes.core.v1 import Namespace
 from pulumi_kubernetes.yaml import ConfigFile
 
-# TODO(levi): Use autonamed Namespace to avoid collisions in CI. Requires YAML transformation support.
+ns = Namespace("ns")
 
-cf = ConfigFile(
+
+def add_namespace(obj):
+    if "metadata" in obj:
+        obj["metadata"]["namespace"] = ns.metadata["name"]
+    else:
+        obj["metadata"] = {"namespace": ns.metadata["name"]}
+
+    return obj
+
+
+cf_local = ConfigFile(
     "yaml-test",
     "manifest.yaml",
+    transformations=[add_namespace],
+)
+
+# Create resources from standard Kubernetes guestbook YAML example in the test namespace.
+cf_url = ConfigFile(
+    name="guestbook",
+    file_id="https://raw.githubusercontent.com/pulumi/pulumi-kubernetes/master/tests/examples/yaml-guestbook/yaml"
+            "/guestbook.yaml",
+    transformations=[add_namespace],
```

Comment:
I believe we are not verifying that this namespace is actually added in the test harness? Can we do that?